src/operator/subgraph/dnnl/dnnl_fc.cc

```diff
@@ -152,6 +153,26 @@ void SgDNNLFCOp::Forward(const OpContext& ctx,
             dnnl::reorder(*in_dnnl_mem, *tmp_mem),
             {{DNNL_ARG_FROM, *in_dnnl_mem}, {DNNL_ARG_TO, *tmp_mem}});
         output = NDArray(tmp_mem);
+      } else if (in_data[idx.sum].dtype() == mshadow::kUint8 &&
+                 out_data[out_index].dtype() == mshadow::kInt8) {
+        auto sum_mem_desc           = in_dnnl_mem->get_desc();
+        auto out_dtype              = get_dnnl_type(mshadow::kInt8);
+        sum_mem_desc.data.data_type = static_cast<dnnl_data_type_t>(out_dtype);
+        dnnl_mem_ptr tmp_mem(new dnnl::memory(
+            sum_mem_desc, CpuEngine::Get()->get_engine(), out_dnnl_mem->get_data_handle()));
+        DNNLStream::Get()->RegisterMem(tmp_mem);
+        const float u8_reorder_scale     = 0.5;
```

Comment:
You can define 0.5 as const or macro in this file and use it in both places - to make it clear for the reader